src/Tizen.Sensor/Tizen.Sensor/BatchData/HeartRateMonitorBatchData.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.Sensor.BatchData
+{
+    /// <summary>
+    /// The HeartRateMonitorBatchData class is used for storing value of HeartRateMonitorBatch sensor.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class HeartRateMonitorBatchData : BatchData
+    {
+        /// <summary>
+        /// Initializes a new instance of the HeartRateMonitorBatchData class.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <param name="timeStamp">Timestamp of HeartRateMonitorBatch sensor event.</param>
+        /// <param name="accuracy">Sensor accuracy of HeartRateMonitorBatch sensor event.</param>
+        /// <param name="state">Value indicating the status of the HeartRateMonitorBatch sensor.</param>
+        /// <param name="heartRate">heartRate of HeartRateMonitorBatch sensor event.</param>
+        /// <param name="rri">rri of HeartRateMonitorBatch sensor event.</param>
+        public HeartRateMonitorBatchData(TimeSpan timeStamp, SensorDataAccuracy accuracy, HeartRateMonitorBatchState state, int heartRate, int rri) : base(timeStamp, accuracy)
+        {
+            State = state;
+            HeartRate = heartRate;
+            Rri = rri;
+        }
+
+        /// <summary>
+        /// Gets the value of the HeartRateMonitorBatch state.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <value> State </value>
+        public HeartRateMonitorBatchState State { get; }
+
+        /// <summary>
+        /// Gets the value of the heart beat per minute.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <value> HeartRate </value>
+        public int HeartRate { get; }
+
+        /// <summary>
+        /// Gets the value R wave-to-R wave interval(RRI) from ECG sensor.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <value> Rri </value>
+        public int Rri { get; }
```

Comment:
How about RRInterval , please explain about unit of RRI. is it milisecond? 
```suggestion
        public int RRInterval{ get; }
```